spec/api-process-spec.js

```diff
@@ -73,6 +74,16 @@ describe('process module', () => {
     });
   });
 
+  describe('process.getWindowsVersionType()', () => {
```

Comment:
```suggestion
  ifdescribe(process.platform === 'win32')('process.getWindowsVersionType()', () => {
```

& then drop the `if`s on the individual tests

Reply:
The second test checks if it throws on non-Windows platforms, though. I copied the `ifit` pattern from other tests that tested platform-specific functionality.🤔